### PR TITLE
Chat list and contact list UI fixes (#291, #274, #275)

### LINKED ISSUE #274: "New group chat" menu item in ios
Instead of plus button "New group chat" button should be implemented in ios version, as it is shown in design:
https://app.zeplin.io/project.html#pid=577681c84d0ec563679d7b51&sid=5798d04694b90980580947c4

### LINKED ISSUE #275: "New Contact" button in ios
"New contact" button should be placed inside the toolbar in ios version https://app.zeplin.io/project.html#pid=577681c84d0ec563679d7b51&sid=57768ac993ec3c6267a0aa14

### LINKED ISSUE #291: Chat screen does not look like design
chat-screen

- [ ] in many designs the top bar colour does not change with header, instead it is semi-transparent black, i saw @alwx do this correctly in past, we should make this uniform
- [x] dates and messages unread are missing? 
- [x] horizontal line seperator between chat sessions is missing
- [x] there should be grey empty area beneath end of chat screen if screen real estate is available

### PR DESCRIPTION
**Pull request by alwx**
fixes #291
fixes #274
fixes #275

Transparent StatusBar on Android is a separate RN-specific feature, that most probably will be merged to React Native soon (https://github.com/facebook/react-native/pull/10185).

### CODE REVIEW

**Comment by rasom** [comment]
As far as i understand the date of the last message should be there.

**Comment by alwx** [comment]
Well, for whatever reason we don't add a timestamp, but this timestamp will be loaded from the database next time you open the app. It is not a UI problem, but I will take a look.

**Comment by rasom** [comment]
This check mark should be shown when the last message in chat was sent by user and it is already seen:

**Comment by rasom** [comment]
#274 is ready

#275 looks like it is not implemented:

**Comment by rasom** [comment]
Wrong icon here:

**Comment by alwx** [comment]
Fixed, squashed, pushed.

**Comment by rasom** [comment]
There is no "seen" check mark in the chats list:



but

**Comment by rasom** [comment]
There is no "seen" check mark in the chats list. It is there if you reload account, but when you receive `seen` message it doesn’t appear in the chats list:



but

**Comment by rasom** [comment]
#275 is ready

File: src/status_im/accounts/handlers.cljs [hunk 1]
```diff
@@ -94,7 +94,8 @@
 
 (defn set-current-account
   [{:keys [accounts] :as db} [_ address]]
-  (let [key (:public-key (accounts address))]
+  (let [key (:public-key (accounts address))
+        _ (.log js/console "ALWX PK" key)]
```
Comment (rasom) [inline src/status_im/accounts/handlers.cljs]: missed debug message

**Comment by alwx** [comment]
@rasom rebased, squashed and pushed :)

**Comment by alwx** [comment]
Rebased.

**Pull request merged**